submissions/krylenger/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,1342 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body class="body">
+    <main class="container-half-screen container-half-screen--film-info">
+      <header class="film-info__header">
+        <h1 class="film-info__title">eX machina</h1>
+        <h2 class="film-info__subtitle">Drama, Sci-Fi, Thriller</h2>
+        <h2 class="film-info__rate">9.4</h2>
+      </header>
```

Comment:
As I understand `.film-info__header` is a BEM element. But according to BEM methodology BEM element can't be outside his parent BEM block. I can't see the parent BEM block.
Also, If you use a couple of headings group them by `hgroup` tag, but in your case better wrap content in `p` instead of using `h2`  tag.

Reply:
✅ Done